src/main/java/org/jetbrains/research/kotlincodesmelldetector/utils/PsiUtils.java

```diff
@@ -0,0 +1,131 @@
+package org.jetbrains.research.kotlincodesmelldetector.utils;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.roots.ProjectFileIndex;
+import com.intellij.psi.*;
+import org.apache.commons.lang.ArrayUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.kotlin.psi.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class PsiUtils {
```

Comment:
Seems redundant for now

Reply:
Why? I mean, that was literally the class the was asked for me to port.